src/main/scala/io/iohk/ethereum/domain/BlockchainReader.scala

```diff
@@ -86,6 +69,18 @@ class BlockchainReader(
     */
   def getReceiptsByHash(blockhash: ByteString): Option[Seq[Receipt]] = receiptStorage.get(blockhash)
 
+  /** get the current best stored branch */
+  // FIXME this should not be an option as we should always have the genesis
+  // but some tests prevent it to simply be BlockchainBranch for now
+  def getBestBranch(): Option[BlockchainBranch] =
```

Comment:
As mentioned in our discusssion, I'm hoping that this method will be part of Consensus soon